subsys/net/l2/openthread/openthread.c

```diff
@@ -277,6 +277,51 @@ int openthread_send(struct net_if *iface, struct net_pkt *pkt)
 	return len;
 }
 
+static inline bool is_valid_ipv6_address(struct net_if_addr *addr)
+{
+	return (addr->is_used && addr->address.family == AF_INET6);
+}
```

Comment:
Given this is used in the loop through the list of IPv6 addresses of an interface, I doubt that they will be of an address family different than `AF_INET6` if `is_used` is true. That said probably better to be safe than sorry.

Reply:
Yeah I could probably remove this function entirely and just check is_used.